doc.php

```diff
@@ -35,26 +35,28 @@
   }
   if (!isset($repository))
     $repository = 'omichel';
+  $url = "https://raw.githubusercontent.com/$repository/webots-doc/";
```

Comment:
I wouldn't define a variable like this (used only once, defined far away from its usage) and revert to the original version.